MirrorProvider/MirrorProvider.Linux/LinuxFileSystemVirtualizer.cs

```diff
@@ -20,6 +21,10 @@ public override bool TryConvertVirtualizationRoot(string directory, out string e
 
         public override bool TryStartVirtualizationInstance(Enlistment enlistment, out string error)
         {
+            string storageRoot = Path.Combine(enlistment.DotMirrorRoot, "src");
```

Comment:
I think it'll be confusing to have this folder called `src`. How about `.storage`? Or some other appropriate "dot" name.

Reply:
OK ... we choose .gvfs/src (and by extension .mirror/src) in the meeting in Redmond, but I'm agnostic.  Happy to change to .storage if you prefer.

Reply:
We definitely agreed to put it inside the `.gvfs` folder, but we hadn't settled on a name that I can recall. I'm open to suggestions, but calling it `src` seems like it will lead to confusion.

Reply:
Done!